### ISSUE TITLE
Move repo to BWL GitHub organization

### ISSUE THREAD

**Issue by Roberciiik** [K4thos/TGC1e#12]
AFAIK, this modification was a part of Black Wyrm Lair, so it might be a good idea to move it under [Black Wyrm Lair GitHub organization](https://github.com/Black-Wyrm-Lair), so it will increase the mod visibility and accessibility.

**Comment by K4thos** [comment]
sure, if maintainers are ok with it, I'm ready to transfer the repo

**Comment by Roberciiik** [comment]
Hello @K4thos

Thanks for fast reply. I am currently one of the BWL GitHub admins as stated [here on G3](https://www.gibberlings3.net/forums/topic/33199-black-wyrm-lair-modding-authors-baronius-vlad-gave-permission-for-the-infinity-engine-modding-community-to-update-maintain-their-mods/?do=findComment&comment=301378). As you are/were maintaining this mod, I will send you an invitation to the organization as well, so you will still be able to work on the repo without additional steps.

PS. I suggested the same transfer for the other repo on your GH: https://github.com/K4thos/EET_Tweaks/issues/10 if you are willing to. 

Pozdrawiam,
Robert
